modules/pubmaticBidAdapter.js

```diff
@@ -603,42 +612,85 @@ function _createImpressionObject(bid, conf) {
     bidfloorcur: bid.params.currency ? _parseSlotParam('currency', bid.params.currency) : DEFAULT_CURRENCY
   };
 
-  if (bid.params.hasOwnProperty('video')) {
-    var videoData = bid.params.video;
+  if (bid.hasOwnProperty('mediaTypes')) {
+    for (mediaTypes in bid.mediaTypes) {
+      switch (mediaTypes) {
+        case BANNER:
+          bannerObj = {};
+          sizes = bid.mediaTypes.banner.sizes;
+          if (sizes !== undefined && utils.isArray(sizes)) {
+            if (!bid.params.width && !bid.params.height) {
+              bannerObj.w = sizes[0][0];
```

Comment:
let's  add parseInt() on every size: width and height.
I think the current code accepts an array of strings as well in the same format

Reply:
ok.